keras/callbacks.py

```diff
@@ -1003,9 +1003,13 @@ class LambdaCallback(Callback):
         # Plot the loss after every epoch.
         import numpy as np
         import matplotlib.pyplot as plt
-        plot_loss_callback = LambdaCallback(
-            on_epoch_end=lambda epoch, logs: plt.plot(np.arange(epoch),
-                                                      logs['loss']))
+
+        def plot_loss(epoch, logs, losses=[]):
+            losses.append(logs['loss'])
+            plt.plot(np.arange(epoch + 1), losses)
+            plt.show()  # Use the best method to update the plot for your backend
```

Comment:
There's a keras.callbacks.CSVLogger already and it's hard to compete with that. I've updated the example to log JSON lines to a file.